docker/nginx/nginx.conf

```diff
@@ -26,6 +26,13 @@ server {
         proxy_pass http://payments-service;
         proxy_set_header Host $host;
     }
+
+    # mozilla/payments-ui static files.
+    # This is aliased to the path mounted from the host
+    # under nginx in the docker-compose.yml.
```

Comment:
good call on explaining where this path comes from. While you're in here, could you add a similar comment about the ``/src/payments-example/`` path?